demo/docker/app/conf/application.conf

```diff
@@ -27,7 +27,6 @@ customProcesses {}
 
 flinkConfig {
   restUrl: "http://jobmanager:8081"
-  queryableStateProxyUrl: "taskmanager:9069"
```

Comment:
Let's leave it here. This config uses older configuration format - we'll change it hopefully in near future...